images/users/snapdeal.svg

```diff
@@ -0,0 +1,36 @@
+                        <path d="M474.169521,78.2007556 C476.284542,78.2007556 478.097096,76.6797786 478.097096,74.4475778 C478.097096,70.390884 473.364441,70.9990483 473.364441,64.2039165 L473.364441,4.05680696 C473.364441,1.92766565 471.65196,0.202834686 469.536939,0.202834686 L469.335669,0.202834686 C467.220648,0.202834686 465.509291,1.92766565 465.509291,4.05680696 L465.509291,66.3341903 C465.509291,72.3184364 469.839406,78.2007556 474.169521,78.2007556" id="Fill-15" fill="#333333"></path>
+                        <g id="Group-19" transform="translate(0.000000, 4.909483)">
+                            <mask id="mask-2" fill="white">
+                                <use xlink:href="#path-1"></use>
+                            </mask>
+                            <g id="Clip-18"></g>
+                            <path d="M21.7180399,90.2297245 C22.8255869,92.1572769 24.034331,92.8673682 26.2494249,92.8673682 L67.5367512,92.8673682 C69.7529695,92.8673682 70.9605892,92.2580713 72.3706033,90.4324459 L103.58881,49.8609787 C104.292692,48.948166 103.487613,47.427189 102.380066,47.427189 L52.3313122,47.427189 C50.1162183,47.427189 48.9074742,46.7170977 47.7999272,44.7906778 L22.6243169,0.973402637 C22.020507,-0.0413370521 20.409223,0.162516903 20.0055587,1.17612407 L0.571201878,48.5427231 C-0.335075117,50.672997 -0.133805164,52.0931795 0.973741784,54.0195994 L21.7180399,90.2297245 Z" id="Fill-17" fill="#E40046" mask="url(#mask-2)"></path>
+                        </g>
+                        <path d="M73.6799824,42.8025355 C74.7875293,44.7300879 75.9962735,45.4401792 78.2113674,45.4401792 L102.379504,45.4401792 C103.588248,45.4401792 104.494525,43.9180696 103.890715,42.8025355 L80.8301256,2.63764368 C79.7225786,0.710091278 78.5138345,0 76.2987406,0 L30.1764378,0 C28.9688181,0 28.0625411,1.52097701 28.6663509,2.63764368 L40.6503462,23.5315416 C41.7578932,25.4579615 42.9655129,26.1680527 45.1817312,26.1680527 L64.2135481,26.1680527 L73.6799824,42.8025355 Z" id="Fill-20" fill="#E40046"></path>
+                    </g>
+                </g>
+            </g>
+        </g>
+    </g>
+</svg>
```

Comment:
Could you please run [svgo](https://jakearchibald.github.io/svgomg/) on your SVG to optimize it.